caas/kubernetes/clientconfig/export_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package clientconfig
+
+var (
+	NewK8sClientSet                               = newK8sClientSet
+	EnsureJujuAdminServiceAccount                 = ensureJujuAdminServiceAccount
+	EnsureClusterRole                             = ensureClusterRole
+	EnsureServiceAccount                          = ensureServiceAccount
+	EnsureClusterRoleBinding                      = ensureClusterRoleBinding
+	GetServiceAccountSecret                       = getServiceAccountSecret
+	ReplaceAuthProviderWithServiceAccountAuthData = replaceAuthProviderWithServiceAccountAuthData
+)
+
+const (
+	AdminNameSpace             = adminNameSpace
```

Comment:
Generally for tests we prefer to use string literals. That way, if the code changes, then the test also needs to be updated.

Reply:
agreed

Reply:
yes.